core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay

```diff
@@ -1,6 +1,7 @@
 # numaresources-operator
-registry.ci.openshift.org/ocp-kni/numaresources-operator:numaresources-operator quay.io/openshift-kni/numaresources-operator:4.15.999-snapshot
-registry.ci.openshift.org/ocp-kni/numaresources-operator:numaresources-operator-tests quay.io/openshift-kni/numaresources-operator-tests:4.15.999-snapshot
+registry.ci.openshift.org/ocp-kni/numaresources-operator:numaresources-operator-tests quay.io/openshift-kni/numaresources-operator-tests:4.16.999-snapshot
```

Comment:
numaresources-operator-tests is always just for the latest branch ?

Reply:
yes